test/log-load.spec.js

```diff
@@ -837,105 +837,105 @@ Object.keys(testAPIs).forEach((IPFS) => {
+          const a = await fromEntry(ipfs, testIdentity, last(items1),
             { length: -1 })
-          assert.strictEqual(a.length, amount)
-          assert.strictEqual(a.values[0].hash, items1[0].hash)
+          strictEqual(a.length, amount)
+          strictEqual(a.values[0].hash, items1[0].hash)
         })
 
         it('returns all entries - including excluded entries', async () => {
           // One entry
-          const a = await Log.fromEntry(ipfs, testIdentity, last(items1),
+          const a = await fromEntry(ipfs, testIdentity, last(items1),
             { length: -1, exclude: [items1[0]] })
-          assert.strictEqual(a.length, amount)
-          assert.strictEqual(a.values[0].hash, items1[0].hash)
+          strictEqual(a.length, amount)
+          strictEqual(a.values[0].hash, items1[0].hash)
 
           // All entries
-          const b = await Log.fromEntry(ipfs, testIdentity, last(items1),
+          const b = await fromEntry(ipfs, testIdentity, last(items1),
             { length: -1, exclude: items1 })
-          assert.strictEqual(b.length, amount)
-          assert.strictEqual(b.values[0].hash, items1[0].hash)
+          strictEqual(b.length, amount)
+          strictEqual(b.values[0].hash, items1[0].hash)
         })
 
         it('respects timeout parameter', async () => {
           const e = last(items1)
           e.hash = 'zdpuAwNuRc2Kc1aNDdcdSWuxfNpHRJQw8L8APBNHCEFXbogus'
           const timeout = 500
           const st = new Date().getTime()
-          const log = await Log.fromEntry(ipfs, testIdentity, e, { timeout })
+          const log = await fromEntry(ipfs, testIdentity, e, { timeout })
           const et = new Date().getTime()
-          assert.strictEqual((et - st) >= (timeout - 10), true, '' + (et - st) + ' should be greater than timeout ' + timeout)
-          assert.strictEqual(log.length, 1)
-          assert.deepStrictEqual(log.values.map(e => e.payload), [e.payload])
+          strictEqual((et - st) >= (timeout - 10), true, '' + (et - st) + ' should be greater than timeout ' + timeout)
+          strictEqual(log.length, 1)
+          deepStrictEqual(log.values.map(e => e.payload), [e.payload])
         })
       })
     })
 
     describe('Backwards-compatibility v0', () => {
-      const entries = [v0Entries.hello, v0Entries.helloWorld, v0Entries.helloAgain]
+      const entries = [hello, helloWorld, helloAgain]
       before(async () => {
-        await Promise.all(entries.map(e => io.write(ipfs, Entry.getWriteFormat(e), Entry.toEntry(e), { links: Entry.IPLD_LINKS })))
+        await Promise.all(entries.map(e => io.write(ipfs, getWriteFormat(e), toEntry(e), { links: IPLD_LINKS })))
       })
 
       it('creates a log from v0 json', async () => {
-        const headHash = await io.write(ipfs, 'dag-pb', Entry.toEntry(v0Entries.helloAgain), { links: Entry.IPLD_LINKS })
+        const headHash = await io.write(ipfs, 'dag-pb', toEntry(helloAgain), { links: IPLD_LINKS })
```

Comment:
Could prolly do the same here as for all other functions for consistency, ie. `await write(...)`. Applies to all other places where io.write/read are used.

Reply:
I notice that io is exported by utils/index.js. Would it make more sense to use the orbit-db-io module directly and destruct the functions?

E.g.

```
import { read, write } from 'orbit-db-io'
```

Reply:
Have opted for destructing read, write from orbit-db-io. Seems more elegant and one less place to go looking for the export.